provider/docker/task.go

```diff
@@ -224,6 +225,43 @@ func (p *Provider) RunCommand(ctx context.Context, id string, command []string)
 	return stdout.String(), stderr.String(), execInspect.ExitCode, nil
 }
 
+func (p *Provider) RunCommandWhileStopped(ctx context.Context, id string, definition provider.TaskDefinition, command []string) (string, string, int, error) {
```

Comment:
It's generally better to return structs, or name the return variables when they go above like 2 or 3.  

I would do one of those.  Otherwise it can be confusing to reason what is being returned from this func.

Godoc comment would help too